morf-core/src/main/java/org/alfasoftware/morf/sql/SelectStatement.java

```diff
@@ -42,77 +44,104 @@
  * <p>The class structure imitates the end SQL and is structured as follows:</p>
  *
  * <blockquote><pre>
- *   new SelectStatement([field], ...)
- *        |----&gt; .from([table])                               = SELECT [fields] FROM [table]
- *                |----&gt; .innerJoin([table], [criterion])     = SELECT [fields] FROM [table] INNER JOIN [table] ON [criterion]
- *                |----&gt; .leftOuterJoin(...)
- *                |----&gt; .where([criterion])                  = SELECT [fields] FROM [table] WHERE [criterion]
- *                |----&gt; .orderBy([fields])                   = SELECT [fields] FROM [table] ORDER BY [fields]
- *                |----&gt; .groupBy([fields])                   = SELECT [fields] FROM [table] GROUP BY [fields]
- *                        |----&gt; having([criterion])          = SELECT [fields] FROM [table] GROUP BY [fields] HAVING [criterion]
- *                |----&gt; .union([SelectStatement])            = SELECT [fields] FROM [table] UNION [SelectStatement]
- *                |----&gt; .unionAll([SelectStatement])         = SELECT [fields] FROM [table] UNION ALL [SelectStatement]
- *  </pre></blockquote>
- *
- * <p>This class does not accept string references to field or table names. Instead, you must provide
- * the methods with a {@link TableReference} or {@link FieldReference} reference.</p>
- *
- * <p>Each method of this class will return an instance of the SelectStatement class. However, this will always
- * be the same instance rather than a new instance of the class.</p>
+ *  SelectStatement.select([field], ...)
+ *    .from([table])                       = SELECT [fields] FROM [table]
+ *    .innerJoin([table], [criterion])     = SELECT [fields] FROM [table] INNER JOIN [table] ON [criterion]
+ *    .leftOuterJoin(...)
+ *    .where([criterion])                  = SELECT [fields] FROM [table] WHERE [criterion]
+ *    .orderBy([fields])                   = SELECT [fields] FROM [table] ORDER BY [fields]
+ *    .groupBy([fields])                   = SELECT [fields] FROM [table] GROUP BY [fields]
+ *      .having([criterion])               = SELECT [fields] FROM [table] GROUP BY [fields] HAVING [criterion]
+ *    .union([SelectStatement])            = SELECT [fields] FROM [table] UNION [SelectStatement]
+ *    .unionAll([SelectStatement])         = SELECT [fields] FROM [table] UNION ALL [SelectStatement]
+ *    .build()</pre></blockquote>

```

Comment:
Hmmm, hasn't this reformatting lost some of the flow? It wasn't supposed to imply that each call was chained, but now it looks like they are.

Reply:
Not sure I follow?